.cicdtemplate/.bitrise/bitrise.yml

```
@@ -0,0 +1,224 @@
+---
+format_version: '8'
+default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
+project_type: android
+trigger_map:
+  - push_branch: develop
+    workflow: develop
+
+  - push_branch: release/*
+    workflow: release
+
+  - push_branch: master
+    workflow: master
+
+  - push_branch: '*'
+    workflow: primary
+
+  - pull_request_source_branch: '*'
+    workflow: primary
+
+# Work-flows definition
+workflows:
+  # Workflow on * branch
```

maybe any branch sounds better 🤔
```suggestion
  # Workflow on any branch
```

---

@minhnimble By using `*`, we have the best explanation of the workflow that will be executed on the `all` branch 😄